src/main.cpp

```diff
@@ -14,45 +16,38 @@ static constexpr auto TASK0_PRIO = 5;
 static constexpr auto TASK1_PRIO = 10;
 static constexpr auto TASK2_PRIO = 20;
 
-void
-task0(void)
+void task0(void)
 {
-    nrf_gpio_cfg_output(LED_COL1);
-    int i = 0;
+    edge::drivers::GPIOPin col1(LED_COL1, edge::drivers::GPIODirection::OUT);
     while (1) {
-        if (i++ % 300000 == 0)
-            nrf_gpio_pin_toggle(LED_COL1);
+        col1.toggle();
     }
 }
 
 // Demonstrates priority change
-void
-task1(void)
+void task1(void)
 {
     edge::userlib::change_priority(2);
-
-    nrf_gpio_cfg_output(LED_COL2);
-    int i = 0;
+    edge::drivers::GPIOPin col2(LED_COL2, edge::drivers::GPIODirection::OUT);
+    col2.set();
     while (1) {
-        if (i++ % 300000 == 0)
-            nrf_gpio_pin_toggle(LED_COL2);
+        col2.toggle();
```

Comment:
keep the i++%30000 or add a yield so we can actually see it changing

Reply:
i mean task 2 is yileding?

Reply:
i dont see it? am i tweaking

Reply:
![image](https://github.com/user-attachments/assets/ce287b87-59f9-4565-8ccf-ea849c00656f)
?

Reply:
oh mb i meant task1, why not have both yield

Reply:
we could do it, doesnt really matter idt?